department-of-veterans-affairs/vets-website
# Ticket #69615 (10-10EZR): Adds a Temporary Fix for Insurance Validation Issue

## Summary

Fixes an issue where the user was not able to submit their 10-10EZR application form due to a schema validation error.

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/69615

## Testing done

1. In `src/applications/ezr/components/IntroductionPage/SaveInProgressInfo.jsx`, set line `33` to `false`.
2. Visit the introduction page and click on the link "Start your application without signing in".
3. Fill out form as normal until getting to insurance section (Take the minimal happy path).
4. Add an insurance policy and test field validation.
5. Continue to Review page.
6. Ensure you can submit the form with a frontend error.

### Quality Assurance & Testing


### Error Handling


### Authentication


### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:


## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
